### PR TITLE
adds extra_label to all import apis

### PR DESCRIPTION
**Pull request by f41gh7**
changes priority for extra_label - now it has priority over original labels

### CODE REVIEW

**Review by valyala — CHANGES_REQUESTED** [review]
Could you update this pull request in accordance to new rules about tags de-duplication outlined in the commit c8ea697db8be2e427b5c1d0af5a67495d55cafda ?

Could you also update `app/vmagent` code as well?

File: app/vminsert/csvimport/request_handler.go [hunk 1]
```diff
@@ -40,14 +40,14 @@ func insertRows(rows []parser.Row, extraLabels []prompbmarshal.Label) error {
 		r := &rows[i]
 		ctx.Labels = ctx.Labels[:0]
 		ctx.AddLabel("", r.Metric)
-		for j := range r.Tags {
-			tag := &r.Tags[j]
-			ctx.AddLabel(tag.Key, tag.Value)
-		}
 		for j := range extraLabels {
```
Comment (valyala) [inline app/vminsert/csvimport/request_handler.go:43]: `extraLabels` must be added after adding user-provided labels, so they could override user-provided labels according to the updated `MetricName.sortTags` functionality at c8ea697db8be2e427b5c1d0af5a67495d55cafda .

Comment (f41gh7) [reply app/vminsert/csvimport/request_handler.go]: updated

Comment (f41gh7) [reply app/vminsert/csvimport/request_handler.go]: yep, changed. Also added test cases.

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
Thanks!

**Pull request merged**